### PR TITLE
Fix image rendering in samples browser

### PR DESCRIPTION
**Pull request by weikanglim**
Our samples in the sample browser aren't rendering certain images. [Example](https://learn.microsoft.com/en-us/samples/azure-samples/todo-nodejs-mongo-terraform/todo-nodejs-mongo-terraform/)

This change removes the image elements in favor of markdown image reference which doesn't allow custom sizing. The UX experience is pretty similar and has been reviewed by @savannahostrowski.

This change also adds frontmatter for the new `aks` template

Example for `nodejs-mongo-swa-func`: [Before](https://github.com/Azure-Samples/todo-python-mongo-aca/tree/main), [After](https://github.com/Azure-Samples/todo-python-mongo-aca/tree/pr/1780)

### CODE REVIEW

**Comment by weikanglim** [comment]
@savannahostrowski For quick context, our samples in the sample browser aren't rendering certain images. [Example](https://learn.microsoft.com/en-us/samples/azure-samples/todo-nodejs-mongo-terraform/todo-nodejs-mongo-terraform/)

I tried to remove the image elements in favor of markdown image reference for simplicity, but it does end up resizing the image and affecting UX. You can check the changes using the `repoman` provided links above. Example for `nodejs-mongo-swa-func`: [Before](https://github.com/Azure-Samples/todo-python-mongo-aca/tree/main), [After](https://github.com/Azure-Samples/todo-python-mongo-aca/tree/pr/1780)

The alternative is to fully-qualify the image URLs, which requires us to specify:

`Screenshot of deployed ToDo app`

instead of

`Screenshot of deployed ToDo app`

**Comment by seesharprun** [comment]
We are noticing this issue with other samples too: https://learn.microsoft.com/en-us/samples/azure-samples/todo-java-mongo-aca/todo-java-mongo-aca/

**Review by wbreza — APPROVED** [review]

File: templates/todo/projects/nodejs-mongo-aks/README.md [hunk 1]
```diff
@@ -7,7 +28,7 @@ A complete ToDo application that includes everything you need to build, deploy,
 
 Let's jump in and get the ToDo app up and running in Azure. When you are finished, you will have a fully functional web app deployed on Azure. In later steps, you'll see how to setup a pipeline and monitor the application.
 
-<img src="assets/resources.png" width="75%" alt="Screenshot of deployed ToDo app">
+!["Screenshot of deployed ToDo app"](assets/resources.png)
```
Comment (weikanglim) [inline templates/todo/projects/nodejs-mongo-aks/README.md:31]:
```suggestion
!["Screenshot of deployed ToDo app"](assets/web.png)
```

**Review by vhvb1989 — APPROVED** [review]

**Comment by weikanglim** [comment]
Given that the last change was a README image rename, probably isn't worth the effort of trying to fix all the branches in the template repository to allow the validation to run. I'm going to override enforcer for this small edit.

**Pull request merged**